pkg/ebpftracer/c/headers/common/context.h

```diff
@@ -197,6 +197,9 @@ statfunc int init_tailcall_program_data(program_data_t *p, void *ctx)
         return 0;
     }
 
+    // TODO(patrick.pichler): this is just a quick fix. The proper fix comes later.
```

Comment:
When is later? Is it part of this change or not?

Reply:
In a later PR, this is a bigger rework.